src/components/tabController/useImperativeTabControllerHandle.ts

```diff
@@ -0,0 +1,15 @@
+import React, {useImperativeHandle} from 'react';
+
+export interface TabControllerImperativeMethods {
+  setTab: (index: number) => void;
+}
+
+const useImperativeTabControllerHandle = (ref: React.Ref<any>, setCurrentIndex: (index: number) => void) => {
```

Comment:
```suggestion
const useImperativeTabControllerHandle = (ref: React.Ref<TabControllerImperativeMethods>,
  setCurrentIndex: (index: number) => void) => {
```

This obligates `useImperativeHandle` to return the interface.